platform/pulpcore/app/serializers/base.py

```diff
@@ -144,6 +143,39 @@ def _populate_generic_fields(self, instance, field_mappings):
             field.mapping.replace(mapping)
 
 
+class NestedModelSerializer(ModelSerializer):
+    """
+    This class allows the serializer to have WritableNestedUrlRelated fields, which represent
+    nested parent objects that are implied by the url of a request.
+    """
+
+    def to_internal_value(self, *args, **kwargs):
```

Comment:
In most cases it's a much better practice to match the specific signature when overriding methods.
```
def to_internal_value(self, data)
```